src/ChurchCRM/dto/Photo.php

```diff
@@ -191,8 +191,8 @@ private function getGDImage($sourceImagePath): \GdImage
 
     private function ensureThumbnailsPath(): void
     {
-        if (!file_exists($this->thubmnailPath)) {
-            mkdir($this->thubmnailPath);
+        if (!file_exists($this->thumbnailPath)) {
```

Comment:
this should probably check if the path is a directory instead of file or directory  

```suggestion
        if (!is_dir($this->thumbnailPath)) {
```

Reply:
Feel free to preemptively file the issue and link it here!